src/requestHandlers/panel.ts

```diff
@@ -0,0 +1,92 @@
+import { Request, Response } from 'express';
+import * as panelService from '../services/panelService';
+import { assertArgumentsDefined, sanitizeResponse as sanitizeResponse } from './utils';
+
+/**
+ * Creates a panel
+ * @param image //image path
+ * @param index //panel index
+ * @param panel_set_id //id of the panel set the panel is a part of
+ * @returns response or genericErrorResponse
+ */
+const _createPanelController = async (image: string, index: number, panel_set_id: number) => {
+    try {
+        return await panelService.createPanel({
+            image:        image,
+            index:        index,
+            panel_set_id: panel_set_id,
+        });
+    }
+    catch (err) {
+        return err as Error;
+    }
```

Comment:
Simpler/equivalent version:
```suggestion
    return panelService.createPanel({
        image,
        index,
        panel_set_id
    });
```

Reply:
should I move the try catch to the upper layer then?